doc/en/example/reportingdemo.rst

```diff
@@ -482,8 +482,9 @@ get on the terminal - we are working on that):
             s = "123"
             g = "456"
     >       assert s.startswith(g)
-    E       assert <built-in method startswith of str object at 0xdeadbeef>('456')
-    E        +  where <built-in method startswith of str object at 0xdeadbeef> = '123'.startswith
+    E       assert False
```

Comment:
Is this a regression?  I seem to remember code in rewrite to strip out this `False` bit.